package.json

```diff
@@ -74,7 +73,7 @@
     "@opengovsg/ng-file-upload": "^12.2.15",
     "@opengovsg/sgid-client": "1.0.4",
     "@opengovsg/spcp-auth-client": "^1.4.20",
-    "@sentry/browser": "^7.20.1",
+    "@sentry/browser": "^7.17.3",
```

Comment:
why are we rolling this (and other dependencies) back?

Reply:
shoot i forgot we already had other dep upgrades, thanks for catching, i'll remerge